spec/tracks_checker_spec.rb

```diff
@@ -147,6 +154,17 @@ module Danger
         end
       end
 
+      def allow_diff_match_(modified_files:, diff_line:, expect_match:)
```

Comment:
Is it intentional that this method's name ends with `_`? Seems a bit odd-looking 🙃

Reply:
Thanks, that was probably a typo + `Replace all` 🙃 Fixed on 21992a4bed538f2a06e465af8c1e81e5fe245b47